crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -290,12 +290,24 @@ impl AdminInterfaceApi {
                 Ok(AdminResponse::NetworkStatsDumped(stats))
             }
             AddAgentInfo { agent_infos } => {
-                self.conductor_handle.add_agent_infos(agent_infos).await?;
+                let mut parsed = Vec::with_capacity(agent_infos.len());
+                for info in agent_infos {
+                    parsed.push(kitsune2_api::AgentInfoSigned::decode(
+                        &kitsune2_core::Ed25519Verifier,
+                        info.as_bytes(),
+                    )?);
```

Comment:
How about instead of short-circuiting here, log parsing errors and carry on?

Reply:
I don't think ignoring errors is a good thing, no. Whatever dev is integrating with conductor would rather get an error to know they did something wrong, than to have something partially work and have to debug why it wasn't fully working, right?

Reply:
It's not exactly ignoring errors, but agreed.